lib/libc/Kconfig

```diff
@@ -55,8 +55,8 @@ menu "C Library"
 choice LIBC_IMPLEMENTATION
 	prompt "C Library Implementation"
 	default EXTERNAL_LIBC if NATIVE_APPLICATION
+	default PICOLIBC
 	default NEWLIB_LIBC if REQUIRES_FULL_LIBC
-	default PICOLIBC if REQUIRES_FULL_LIBC
 	default MINIMAL_LIBC
```

Comment:
Should the `default MINIMAL_LIBC` be removed with change, or have some other criteria applied?

Reply:
This is still relevant because `MINIMAL_LIBC` should be used if `PICOLIBC` is not supported by the toolchain in use.

Reply:
Currently it's used with ARC MWDT toolchain (in case of no full-size libc is required). So we still need ti have it in the end of the list.